Library/Formula/libassuan.rb

```diff
@@ -18,7 +18,8 @@ class Libassuan < Formula
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
-                          "--prefix=#{prefix}"
+                          "--prefix=#{prefix}",
+                          "--enable-static"
```

Comment:
I think it'd be okay, but all the bottles are `cellar :any` so we can play it better safe than sorry.